content/php/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Data Types'
+Description: 'PHP supports a number of primitive data types.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Types'
+  - 'Integers'
+  - 'Booleans'
+  - 'Strings'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+PHP supports the following primitive data types:
+
+- bool: a value that's either `true` or `false`.
+- int: a whole number value.
+- float: a numeric value with decimal.
+- string: a series of characters.
+- array: an ordered map of key/value pairs.
+- object: an instance of a pre-defined class.
+- callable: a reference to a PHP function.
+- iterable: represents any array or object implementing the `Traversable` interface.
+- resource: a reference to an external resource.
+- NULL: represents a variable with no value.
+
+Rather than being declared in code, PHP typically decides types at runtime depending on context. There are a number of ways to determine the type of a variable or expression.
```

Comment:
```suggestion
Rather than being declared in code, the data type is decided at runtime depending on the context. There are many ways to determine the type of a variable or expression.
```

Reply:
Alternatively (if I misunderstood "a number of"): There are a few ways to determine the type of a variable or expression.